src/main/java/edu/cornell/kfs/concur/service/ConcurAccountValidationService.java

```diff
@@ -7,4 +7,5 @@
     
     ValidationResult validateConcurAccountInfo(ConcurAccountInfo concurAccountInfo);
 
+    ValidationResult validateConcurAccountInfoObjectCodeNotRequired(ConcurAccountInfo concurAccountInfo);
```

Comment:
The acceptance criteria is out of date. I will update it.

For personal corp card expenses there is no object code value in the report details so only the rest of the accounting string will be validated.